data/resources/ui/sidebar-chat-row.ui

```diff
@@ -68,25 +69,41 @@
                   </object>
                 </child>
                 <child>
-                  <object class="GtkLabel">
-                    <property name="hexpand">True</property>
+                  <object class="GtkStack">
                     <property name="halign">end</property>
```

Comment:
I think that this can be deleted after setting last_message_label's hexpand to true, right?

Reply:
You're right.